### DISCUSSION TITLE
Animated location

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#87] [Q&A] [answered]
Do you use the "Animated location" and have reports about it?
It should provide smooth map movement in navigation / follow.

"Settings | Location | Advanced | Animated location" or search settings for "animated".

Note: in all navigators that have smooth and animated map movement,
user location is displayed some distance behind the real GPS location!

_If you want GPS location in real time, then turn off animations and smooth movement in apps._

<img src="https://user-images.githubusercontent.com/3484020/226095846-a81377e0-8125-4ea4-afbb-94bdf9beab9d.jpg" width="50%"/><img src="https://user-images.githubusercontent.com/3484020/226095849-54003e1f-3f27-4c70-97a6-f76c663e9c62.jpg" width="50%"/>

**Comment by DolomitesTom** [comment, marked as answer]
Yes, i use it already since a long time and it works perfect. At the beginning it was a bit lazy, but after the performance update great.
